recipes-extended/dpdk/dpdk.inc

```diff
@@ -16,6 +20,7 @@ COMPATIBLE_HOST_armv6 = 'null'
 
 COMPATIBLE_MACHINE = "(imx|qoriq)"
 
+export INSTALL_PATH = "${prefix}/share/dpdk"
```

Comment:
This is `${localstatedir}` so please use this accordingly. No hardcoded values please.

Reply:
${localstatedir} is /var, dpdk installs tools to /usr/share/dpdk. May I know why to use ${localstatedir}?

Reply:
Yeah, probably ${datadir}

Reply:
`${libexecdir}` might work better.